Spanner/tests/Unit/TransactionConfigurationTraitTest.php

```diff
@@ -103,18 +103,28 @@ public function testConfigureSnapshotOptionsStrong()
         $this->assertTrue($res['readOnly']['strong']);
     }
 
-    public function testConfigureSnapshotOptionsMinReadTimestamp()
+    /**
+     * @dataProvider timestamps
+     */
+    public function testConfigureSnapshotOptionsMinReadTimestamp($timestamp, $expected = null)
     {
-        $args = ['minReadTimestamp' => $this->ts, 'singleUse' => true];
+        $time = $this->parseTimeString($timestamp);
+        $ts = new Timestamp($time[0], $time[1]);
+        $args = ['minReadTimestamp' => $ts, 'singleUse' => true];
         $res = $this->impl->proxyConfigureSnapshotOptions($args);
-        $this->assertEquals(self::TIMESTAMP, $res['readOnly']['minReadTimestamp']);
+        $this->assertEquals($expected ?: $timestamp, $res['readOnly']['minReadTimestamp']);
     }
 
-    public function testConfigureSnapshotOptionsReadTimestamp()
+    /**
+     * @dataProvider timestamps
+     */
+    public function testConfigureSnapshotOptionsReadTimestamp($timestamp, $expected = null)
```

Comment:
The data provider doesn't look like it ever provides `$expected`, do we need it?